### ISSUE TITLE
CMake Error "Target 'pdalcpp' INTERFACE_INCLUDE_DIRECTORIES property contains path: ... LASZIP_INCLUDE_DIR-NOTFOUND"

### ISSUE THREAD

**Issue by ilovezfs** [PDAL/PDAL#1558]
With 1.5.0, and laszip not installed, I get
```
-- Configuring done
CMake Error in CMakeLists.txt:
  Target "pdalcpp" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/tmp/pdal-20170407-14460-fogp4l/PDAL-1.5/LASZIP_INCLUDE_DIR-NOTFOUND"

  which is prefixed in the build directory.
```

I can work around it with
```
      inreplace "CMakeLists.txt", /^        \${LASZIP_INCLUDE_DIR}\n/, ""
```
when building without the laszip option.

See https://bot.brew.sh/job/Homebrew%20Core/20258/

**Comment by ilovezfs** [comment]
gentle ping on this

**Comment by abellgithub** [comment]
Are you building from source?  OS/X?

**Comment by ilovezfs** [comment]
Yes, and yes. Specifically: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pdal.rb

**Comment by abellgithub** [comment]
I'm not having any problem with your formula.  I commented out the inreplace line and then:

`brew install pdal.rb`

Then

`brew uninstall laszip`
`brew uninstall pdal`
`brew install pdal.rb`

No problem.

**Comment by ilovezfs** [comment]
It's probably pouring the bottle? You need to do --build-from-source

**Comment by ilovezfs** [comment]
Gentle ping on this.

**Comment by hobu** [comment]
Is this a patch for Homebrew or PDAL?

**Comment by chambbj** [comment]
FWIW, apart from Homebrew, I can confirm that I get the same CMake error when building without laszip on Ubuntu.

**Comment by ilovezfs** [comment]
@hobu it's a PDAL bug, which we're dancing around in the Homebrew formula currently:
```
    if build.with? "laszip"
      args << "-DWITH_LASZIP=TRUE"
    else
      # CMake error "Target 'pdalcpp' INTERFACE_INCLUDE_DIRECTORIES property
      # contains path: ... LASZIP_INCLUDE_DIR-NOTFOUND"
      # Reported 7 Apr 2017 https://github.com/PDAL/PDAL/issues/1558
      inreplace "CMakeLists.txt", /^        \${LASZIP_INCLUDE_DIR}\n/, ""
      args << "-DWITH_LASZIP=FALSE"
    end
```
https://github.com/Homebrew/homebrew-core/blob/master/Formula/pdal.rb#L28-L36

**Comment by abellgithub** [comment]
I believe this to be fixed in master: 8b5220eda7b728677858000c7122803412a2e5ba
